cli.js

```diff
@@ -985,12 +1056,12 @@ async function init(packageJson, queries, options) {
       return new Promise((res, rej) =>
         ffmpeg()
           .addInput(files.audio.file.path)
-          .audioCodec('aac')
+          .audioCodec(options.format == 'flac' ? 'flac' : 'aac')
```

Comment:
The issue is that musicbrainz wont do anything if not taglib
https://github.com/miraclx/freyr-js/pull/82/files#diff-180b3d2af89416b99563fff8c37eb8ca1ef7591fc1ddd736b5dacf740ccd3760R948

I could make a first PR adding format support or Taglib usage then update this pr to finish musicbrainz integration
Which would you prefer ?

Reply:
I don't understand. What do you mean by Musicbrainz won't do anything if not taglib?

Reply:
Musicbrainz tags are not taggable when using Atomicparsley so I disabled it

Reply:
Ho yes sorry I didn't understand this this way

Reply:
I mean, did you try to use taglib to tag the same file. Before you introduced flac?

Reply:
No I kept default behavior for m4a and used taglib for flac

Reply:
the first thing in that list is Advanced Audio Codec (AAC): AAC, m4a is AAC.

Reply:
I'll try again by looking deeper into it but my initial test with just removing FLAC switch wasn't really successful

Reply:
So I just looked deeper into it and the simplest issue was just that it detected the format as 'm4a' instead of 'audio/aac' but there is still an issue with it that seems linked to ffmpeg
Error : [ADTS audio header not found]